counterpartylib/lib/messages/issuance.py

```diff
@@ -133,7 +176,10 @@ def validate (db, source, destination, asset, quantity, divisible, callable_, ca
             balances = cursor.fetchall()
             cursor.close()
             if util.enabled('numeric_asset_names'):  # Protocol change.
-                if len(asset) >= 13:
+                if subasset_longname is not None and util.enabled('subassets'): # Protocol change.
```

Comment:
hmm, ok. When looking at PEP8, I see the following:

```Also, beware of writing if x when you really mean if x is not None -- e.g. when testing whether a variable or argument that defaults to None was set to some other value. The other value might have a type (such as a container) that could be false in a boolean context!```

Perhaps that is what you are referring to? That's fine I think then, if there is absolutely no way that `subasset_longname` could be a blank string